tutorials/_posts/2021-10-14-vanilla-gan.md

```diff
@@ -198,6 +207,8 @@ function train_gen!(discriminator, generator)
     # the parameters of the generator
     grads = back(1.0f0)
     Flux.update!(opt_gen, Flux.params(generator), grads)
+    # Set discriminator back into train mode
+    trainmode!(discriminator)
```

Comment:
Should dropout be on during inference too? If not, I'd recommend:
```suggestion
    trainmode!(discriminator, mode=:auto)
```

Reply:
bumping this @rkube